tests/xtimer_reset/tests/01-run.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+def testfunc(child):
+    child.expect_exact("This test tests re-setting of an already active timer.")
+    child.expect_exact("It should print three times \"now=<value>\", with "
+                       "values approximately 100ms (100000us) apart.")
```

Comment:
Is it really necessary to expect all of this?

Reply:
no, but does it hurt?

Reply:
Well every line checked is a check you need to remember to change if you change the line.

Reply:
yes, it also hardens the test against miss-interpretation...